pkg/reconciler/intevents/reconciler.go

```diff
@@ -195,97 +185,9 @@ func (psb *PubSubBase) ReconcilePullSubscription(ctx context.Context, pubsubable
 	}
 
 	status.SinkURI = ps.Status.SinkURI
-
-	// The old and new PullSubscriptions can co-exist without any problems. So to bias in favor of
-	// double event delivery over dropped events, don't delete the old one until the new one is
-	// ready.
-	if ps.Status.IsReady() {
-		err = psb.deleteOldPubSubPullSubscription(ctx, pubsubable, ps)
-		if err != nil {
-			return ps, err
-		}
-	}
-
 	return ps, nil
 }
 
-func (psb *PubSubBase) deleteOldPubSubTopic(_ context.Context, pubsubable duck.PubSubable, t *inteventsv1alpha1.Topic) pkgreconciler.Event {
-	// TODO This will be deleted at the same time as the old pubsub.cloud.google.com CRDs. That is
-	// expected to happen after 0.15, before 0.16.
-	oldT, err := psb.pubsubClient.PubsubV1alpha1().Topics(t.Namespace).Get(t.Name, v1.GetOptions{})
-	if apierrs.IsNotFound(err) {
-		// It doesn't exist, so there is nothing to delete.
-		return nil
-	} else if err != nil {
-		return pkgreconciler.NewEvent(corev1.EventTypeWarning, "OldTopicGetFailed", "unable to get old Topic in the `pubsub.events.cloud.google.com` API group: %w", err)
-	}
-	if !v1.IsControlledBy(oldT, pubsubable.GetObjectMeta()) {
-		// If this pubsubable doesn't own it, then just ignore it. Generate an event in case users
-		// are interested, but do not stop reconciliation of pubsubable, nor give it a Ready=false
-		// status.
-		psb.Recorder.Eventf(pubsubable,
-			corev1.EventTypeWarning,
```

Comment:
this remainded me of the runtime.Object in identity spec.
That can be removed now? maybe you did already...

Reply:
I would like to do that in a follow up, as I think that will require real code changes and want to minimize those in this PR, so that they don't get overlooked by accident. I'll send that PR as soon as this one is merged.

Reply:
Never mind, it required no additional code changes. 

Done.

Reply:
sounds good!